test-case/check-userspace-packages.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-packages
+## Preconditions:
+##    N/A
+## Description:
+##    check the userspace packages building and install the packages
+## Case step:
+##    1. download latest alsa-lib, pulseaudio debian packages and UCM
+##    2. install the userspace packages and UCM
+##    3. restart pulseaudio
+## Expect result:
+##    All packages and UCM are installed successfully and pulseaudio restarts successfuly
```

Comment:
This is an installation script, it does not belong to the `test-case` directory. Can you submit this in a different Pull Request?